### PR TITLE
feat: want to write backend Of comment section of blogs page #369

### LINKED ISSUE #369: feat: want to write backend Of comment section of blogs page
image

### PR DESCRIPTION
**Pull request by nishant0708**
**What kind of change does this PR introduce?**

Screenshot 2024-07-30 221940

**Issue Number:**

Fixes #369 
**Snapshots/Videos:**

**Other information**

### CODE REVIEW

**Comment by nishant0708** [comment]
hey @Tanay-ErrorCode  @Doraemon012  please review this whenever possible

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Can you convert this to non-modal one? Down the article, the comments should be shown.

**Comment by nishant0708** [comment]
Yes I guess @Tanay-ErrorCode . Same ui just below content naa.?

**Comment by Tanay-ErrorCode** [comment]
@nishant0708 Yes.

**Comment by nishant0708** [comment]
@Tanay-ErrorCode okay both pr changes will be done by tomorrow

**Comment by nishant0708** [comment]
image
hey @Tanay-ErrorCode done . let me know any further change if required

File: src/components/Article/DiscussionModal.tsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+      const newCommentData: CommentData = {
+        id: newCommentId,
+        author: user.name,
+        avatar: user.pic,
+        content: newComment,
+        timestamp: new Date().toISOString(),
+      };
+
+      // Store comment details in `comments` node
+      const commentRef = ref(db, `comments/${newCommentId}`);
```
Comment (Doraemon012) [inline src/components/Article/DiscussionModal.tsx:140]: @nishant0708 The comments node must be inside the single article and it should only contain the comments for that article only.

* articles
  * {an_article}
    * {content...}
    * comments

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
@nishant0708 Please make the required changes.

**Comment by nishant0708** [comment]
hey @Doraemon012 i did that only in start but it was throwing a lot of error
Screenshot 2024-07-30 220110
see . so Did Normatilazation of Comments

**Comment by nishant0708** [comment]
hey @Tanay-ErrorCode @Doraemon012  fixed this 

Screenshot 2024-08-02 202011

**Review by Doraemon012 — APPROVED** [review]

**Pull request merged**